pkg/eventprocessor/eventfeed/impl/eventfeed_test.go

```diff
@@ -337,3 +343,101 @@ func TestInfura(t *testing.T) {
 		}
 	}
 }
+
+func TestDuplicateEvents(t *testing.T) {
+	t.Parallel()
+
+	dbURI := tests.Sqlite3URI(t)
+	systemStore, err := system.New(dbURI, tableland.ChainID(1337))
+	require.NoError(t, err)
+
+	backend := duplicateEventsChainClient{
+		backends.NewSimulatedBackend(make(core.GenesisAlloc), math.MaxInt64),
```

Comment:
hum is there a need to pass the simulated backed to your mock? I'm not seeing that being used, maybe I'm missing something

Reply:
the only place that was being used is `backend.Blockchain().CurrentHeader().Number.Int64()` to get a block number to pass it to the feed. but we can use a hardcoded block number. so i removed it, it looks cleaner this way. thanks!